src/plans/D3MAaveDaiPlan.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021-2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.8.13;
```

Comment:
wondering if this should be `>=0.6.12` and deployed with 0.8.13 instead, as I think it could break compilation as dependency of `dss-exec-lib` in tests (`DssAction.t.sol`) for the `bar` action.
cc: @brianmcmichael

Reply:
I definitely don't want to hinder progress toward the later solidity version on behalf of dss-exec-lib. We can probably use the rpc tests for this for now until we have a way to manage different solc versions.

Reply:
I hadn't seen this comment before, but as I mentioned in mine, we must not leave a pragma that allows to compile the contract in a broken state and that's what means leaving the `>=0.6.12` one here.
`dss-exec-lib` will need to figure it out how to include this dependency, might be better just a mock or worst case maintain a branch of this repo for it.